lib/Migration/Version2000Date20201208130255.php

```diff
@@ -62,9 +62,8 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 				'notnull' => true,
 				'length' => 1,
 			]);
-			$table->addColumn('message', 'string', [
+			$table->addColumn('message', 'text', [
```

Comment:
Could the constants in `OCP\DB\Types` be used instead?

Reply:
yes